test/jit/test_save_load_for_op_version.py

```diff
@@ -425,3 +421,35 @@ def _helper(m, fn):
                 self.assertEqual(mr, hr)
 
         _helper(v3_mobile_module, current_mobile_module)
+
+    def test_versioned_linspace(self):
+        scripted_module = torch.jit.load(
+            pytorch_test_dir + "/jit/fixtures/test_versioned_linspace_v7.ptl")
+
+        buffer = io.BytesIO(scripted_module._save_to_buffer_for_lite_interpreter())
```

Comment:
`test_versioned_div_scalar(self, sample_input)` is probably a good example. We want to use the current source module, and save it to buffer, load it, then compare the result with loading the old check in model directly. Basically, compare
```
old_m = load("/jit/fixtures/test_versioned_linspace_v7.ptl) # use upgrader
new_m = load("resave_from_current.ptl) # no upgrader will be used
```